<?xml version="1.0"?>
<launch>
	<arg name="namespace_0"	default="tello_ID0" />

	<group ns="$(arg namespace_0)">
		<node pkg="tello_control" name="pid_control" type="pid_control_node" output="screen">
			<param name="constante_P" value="1.5" />
			<param name="constante_I" value="0.01" />
			<param name="constante_D" value="0.3" />
			<param name="constante_P_Z" value="0.8" />
			<param name="constante_I_Z" value="0.01" />
			<param name="constante_D_Z" value="0.3" />
			<param name="DIST_MAX" value="0.1" />
			<param name="TAG_ID" value="0" />
		</node>
	</group>
</launch>
